site/generate-jenkins-io-template.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+WWW_ROOT_DIR="${WWW_ROOT_DIR:-${1:-./www2}}"
+SELF_URL="${SELF_URL:-${2:-https://updates.jenkins.io}}"
+
+wget -q --convert-links -O "$WWW_ROOT_DIR/index.html" --convert-links https://www.jenkins.io/templates/updates/index.html
```

Comment:
Please use long options in scripts where possible (e.g. `sed` isn't for compatibility) to make them easier to understand.

```suggestion
wget --quiet --convert-links --output-document "$WWW_ROOT_DIR/index.html" --convert-links https://www.jenkins.io/templates/updates/index.html
```